app/views/spree/shared/_login.html.erb

```diff
@@ -16,4 +16,20 @@
       <%= f.submit Spree.t(:login), class: 'btn btn-lg btn-success btn-block', tabindex: 3 %>
     </div>
   </fieldset>
-<% end %>
\ No newline at end of file
+<% end %> -->
+<%= form_for Spree::User.new, as: :spree_user, url: spree.create_new_session_path do |f| %>
+  <%= render :partial => 'spree/shared/sign_in_header' %>
+  <fieldset id="password-credentials">
```

Comment:
Remove the 'fieldset' tag. As i cant see this tag inside template
<img width="1054" alt="Screen Shot 2021-11-08 at 12 45 38 PM" src="https://user-images.githubusercontent.com/78048076/140702914-25f26802-3686-497b-806b-48c46a7e628d.png">